pandas-stubs/_typing.pyi

```diff
@@ -373,4 +373,13 @@ ValidationOptions: TypeAlias = Literal[
     "many_to_many",
     "m:m",
 ]
+
+RandomState = Union[
```

Comment:
Should be `RandomState:  TypeAlias = Union[`

(Sorry I should have caught that before - it's a difference between pandas and the stubs)

Reply:
Done!